README.md

```diff
@@ -26,6 +26,14 @@ Just execute:
 
     $ ./build.js
 
+In the occassion you are running a windows platform, we will have to make a few fixes for your machine to be able to build this app:
+    1. Please make sure you have installed git on your machine.
+    2. Open cmd with administrator privilages (Right click on cmd -> run as administrator).
+    3. Set your current working directory to this folder ( "cd {path root of polypod}/windows") and run the convert-symlinks.js file with node. 
+    4. You should recieve an output saying "All set up!".
+    5. Go back to the root folder and run build.js (cd ..).
```

Comment:
Again, possibly tell them how to do that... A sequence of commands might be enough.